Tools/automation.ps1

```diff
@@ -0,0 +1,790 @@
+                            }
+                        }
+                        locations = @{
+                            value = @(
+                                @{
+                                    locationName     = "West US"
+                                    failoverPriority = 0
+                                    isZoneRedundant  = $false
+                                }
+                                @{
+                                    locationName     = "East US"
+                                    failoverPriority = 1
+                                    isZoneRedundant  = $false
+                                }
+                            )
+                        }
+                        ipRules =@{
+                            value = @(
+                                @{
+                                    ipAddressOrRange = "104.42.195.92"
+                                }
+                                @{
+                                    ipAddressOrRange = "40.76.54.131"
+                                }
+                                @{
+                                    ipAddressOrRange = "52.176.6.30"
+                                }
+                                @{
+                                    ipAddressOrRange = "52.169.50.45"
+                                }
+                                @{
+                                    ipAddressOrRange = "52.187.184.26"
+                                }
+                                @{
+                                    ipAddressOrRange = "0.0.0.0"
+                                }
+                            )
+                        }
+                        backupPolicy = @{
+                            value = @{
+                                type = "Periodic"
+                                periodicModeProperties = @{
+                                        backupIntervalInMinutes        = 240
+                                        backupRetentionIntervalInHours = 720
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            @{  ObjectType = "CosmosDBDatabase"
+                ParameterPath  = "$ParameterFolderPath\cosmosdb-sql.$Index.json"
+                TemplatePath   = "$TemplateFolderPath\CosmosDB\cosmosdb-sql.json"
+                Error      = ""
+                Parameters = @{
+                    '$Schema' = $JSONSchema
+                    contentVersion = $JSONContentVersion
+                    Parameters = @{
+                        cosmosName = @{ value = $CDBAccountName }
+                        sqlname    = @{ value = $CDBDatabaseName }
+                        options    = @{
+                            Value  = @{
+                                autoscaleSettings = @{
+                                    maxThroughput = 4000
```

Comment:
Throughput directly relates to the cost someone will hit running this.  We should probably either lower the default or provide easy options for a user to opt between dedicated resources and a pay as you go consumption plan.

Reply:
+1

Reply:
@jamespik , @arjun-msft - I will require some additional details on what we recommend to meet a minimum requirement.

@denelon - Do we have any documentation on performance expectancies? Any metrics that state "We recommend these values if you have this many users" or "We recommend these values if you have 'x' number of applications?"

Reply:
@RDMacLachlan we haven't performed extensive testing to determine capacity. We are still running some tests with "larger" databases so we can load test and make some of these decisions. Since this has been developed in open-source, we have optimized for getting the reference out over expense. We will be standing up a full clone of the community repository. We're also aware of the relative costs associated with Cosmos DB in shared and dedicated configurations. The intent is to also support other data stores like SQL Server. The big variables are likely to be the size of a private catalog and the responsiveness given a set of users.